ui/src/router/components/form/validation/schema.js

```diff
@@ -364,10 +365,9 @@ const schema = (maxAllowedReplica) => [
         .test("unique-rule-names", validateRuleNames),
       routes: yup
         .array(routeSchema)
-        .required()
```

Comment:
`Array.required` has been deprecated and the same functionality is performed by `min(1, ...)`.